docker/terra-node-zsh/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM cerner/terra-node-base:latest
+
+#install zsh / oh-my-zsh
+RUN apk add zsh curl wget git \
+  && wget https://github.com/robbyrussell/oh-my-zsh/raw/master/tools/install.sh -O - | zsh || true
+
+ENV WDIO_BAIL FALSE
```

Comment:
why not `ENV WDIO_BAIL false`?

Reply:
seems like all caps env variables are in vogue, also i only check for all caps false in the config.

Reply:
I'm not sure I follow... If we set 
```
ENV WDIO_BAIL false
```
can't we remove [this change?](https://github.com/cerner/terra-toolkit/pull/361/files#diff-cbe4c848b6c22e669052e0ba331eeca6R25)

Reply:
oh I see.... nevermind... we want false even though CI is true....

Reply:
i switched it to lowercase false, that seems more normal? i dunno